src/main/java/Lindi.java

```diff
@@ -0,0 +1,227 @@
+        LocalFuncs lf = new LocalFuncs();
+
+        switch (userInputTokens[0]) {
+            case "list":
+                lf.executeList();
+                break;
+            case "unmark":
+            case "mark":
+            case "delete":
+                lf.executeModifyTask(userInput);
+                break;
+            case "todo":
+            case "event":
+            case "deadline":
+                lf.executeCreateTask(userInput);
+                break;
+            case "bye":
+                lf.goodByeAndExit();
+                break;
+            default:
+                System.out.println("Uhh, English please? Haha, just kidding...\n" +
+                        "but for reals I didn't really understand that :(");
+                break;
+        }
+    }
+
+
+    /**
+     * Prints a horizontal line for better readability
+     */
+    private static void printSeparator() {
+        System.out.println("-".repeat(50));
+    }
+
+    /**
+     * Prints a greeting message.
+     */
+    private static void greeting() {
+        String name = "Lindi";  // Log It N Do It -> LINDI
+
+        printSeparator();
+        System.out.println("Hello! I'm " + name);
+        System.out.println("What can I do for you?");
+        printSeparator();
+    }
+
+    /**
+     * Loads the taskList from the data file for this session
+     */
+    private static void loadFromFile() {
+        try {
+            // Creates the directory if it does not exist yet. No effect if it exists.
+            String dataDir = "./.data";
+            File dir = new File(dataDir);
+            dir.mkdir();
+
+            // Creates the file if it does not exist yet. No effect if it exists.
+            File dataFile = new File(dir, "LindiData.txt");
+            Boolean fileJustCreated = dataFile.createNewFile();
+
+            if (fileJustCreated) {
+                // No data to read from to load into taskList
+                return;
+            }
+
+            // Data file exist in folder, load it into our taskList
+            Scanner fileScanner = new Scanner(dataFile);
+            while (fileScanner.hasNextLine()) {
+                String parsedString = fileScanner.nextLine();
+                Task task = Task.createFromParsedString(parsedString);
+                taskList.add(task);
+            }
+        } catch (NullPointerException e) {
+            System.out.println(e);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        } catch (CreateTaskException e) {
+            System.out.println("Error loading tasks from save file. Might be corrupted...");
+            throw new RuntimeException(e.getMessage());
+        }
+    }
+
+    private static void chatLoop() {
+        Scanner scanner = new Scanner(System.in);
+        String userInput;
+        while (true) { // This will not be an infinite loop, because goodByeAndExit() terminates the program when called
```

Comment:
Comment should be indented properly, although trailing comments are allowed, for clarity it might be better to put the comment inside the while loop or just above it